src/Cron/OrderImportCron.php

```diff
@@ -53,6 +53,9 @@ public function handle() {
       $this->loggerContract->debug(TranslationHelper::getLoggerKey('cronStartedMessage'), ['method' => __METHOD__]);
       $this->orderService->process($externalLogs, 1);
     }
+    catch(\Exception $e) {
+      $this->loggerContract->error(TranslationHelper::getLoggerKey('test'), ['additionalInfo'=> $this->orderService->process($externalLogs, 1) ,'message' => $e->message, 'method' => __METHOD__]);
```

Comment:
change to a different logger key (declared as a constant). We don't want to eat this Exception, but `test` can't be the desired logger key.

Reply:
What do you mean by eat?

Reply:
Meaning we want to log the Exception's details, but `test` is not going to be the right key here.

Reply:
Ok thank you for the clarification